sufia-models/lib/sufia/models/stats/user_stat_importer.rb

```diff
@@ -25,9 +30,24 @@ def import
       log_message('User stats import complete.')
     end
 
+    # Returns an array of users sorted by the date of their last
+    # stats update. Users that have not been recently updated 
+    # will be at the top of the array.
+    def users_sorted
+      users = []
+      ::User.find_each do |user|
+        users.push(UserRecord.new(user.id, user.user_key, date_since_last_cache(user)))
+      end
+      users.sort! {|a, b| a.last_stats_update <=> b.last_stats_update}
+    end
 
 private
 
+    def delay 
+      return if @delay_secs.nil?
```

Comment:
You can also coerce `@delay_secs` to an integer on line 9, and strike this line, since if `options[:delay_secs]` is nil, that will ultimately tell Ruby to sleep for 0 seconds, which has the effect of not sleeping.